src/test/java/org/primeframework/mvc/URLPrinter.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package org.primeframework.mvc;
+
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+import java.util.Comparator;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+import com.google.inject.Inject;
+import io.fusionauth.http.HTTPMethod;
+import org.primeframework.mvc.action.ExecuteMethodConfiguration;
+import org.primeframework.mvc.action.config.ActionConfiguration;
+import org.primeframework.mvc.action.config.ActionConfigurationProvider;
+
+/**
+ * Prints out URLs/routes in the app based on Prime MVC actions. To use, do an untargeted binding in your test Guice module
+ * like this - bind(URLPrinter.class);
+ * Then inject URLPrinter into your tests and call the dump method (see below)
+ *
+ * @author Brady Wied
+ */
+public class URLPrinter {
```

Comment:
I wonder if there is a better name for this. I think you are trying to print the "routes" or endpoints managed by the actions resolved by `prime-mvc`?

Reply:
Yes. I wasn't sure of the naming. RoutePrinter might be better?